src/components/BackButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Text, StyleSheet, TouchableOpacityProps } from 'react-native'
+import { useNavigation } from '@react-navigation/native'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+
+type BackButtonProps = {
+  /**
+   * Value back button should display
+   * @default 'Cancel'
```

Comment:
Unfortunately it doesn't. I think its better if I replace it with `@defaultValue` https://github.com/microsoft/tsdoc/issues/27

This is what it looks with `@default` 
![image](https://user-images.githubusercontent.com/44710980/93544438-b2f82580-f923-11ea-81f8-b5fc45df1b36.png)

This is what it looks with `@defaultValue`
![image](https://user-images.githubusercontent.com/44710980/93544457-c1ded800-f923-11ea-9a41-2406d7f89a25.png)

Reply:
And if you remove it completely, can Intellisense pick it up when you're trying to use another component? I see very little value in maintaining a default constant value twice

Reply:
Default isn’t necessary that was just to make it fancy so we don’t have to cmd+click to go to component and see default value. You are right we will end up with maintaining default value twice. I can remove it. 
Intellisense works (thanks to very good TS integration in vscode) if you start typing “la” it will show it has “label” which if you hover will show that.